Tests/APITesters/SwiftAPITester/SwiftAPITester.xcodeproj/project.pbxproj

```diff
@@ -360,7 +360,6 @@
 		A55F62BB26EAFFD200A1B466 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Did you mean to change all these?

Reply:
I did 😄
```
Update to make func a var
Also updated some project settings for the APITesters to produce a couple less warnings
Updated PromotionalOfferAPI.Swift to remove a couple warnings
```

This one change in particular showed up as not inherited, so I removed it and all seemed well.

Reply:
https://developer.apple.com/documentation/xcode/build-settings-reference?changes=_8

```
Always Search User Paths (Deprecated)
Setting name: ALWAYS_SEARCH_USER_PATHS

This setting is deprecated as of Xcode 8.3 and may not be supported in future versions. It is recommended that you disable the setting.
```